extension-style/src/test/java/com/mapbox/maps/extension/style/atmosphere/generated/AtmosphereTest.kt

```diff
@@ -925,6 +925,12 @@ class AtmosphereTest {
     assertNotNull(style.getAtmosphere())
     verify(exactly = 0) { style.setStyleAtmosphere(any()) }
   }
+
+  @Test
+  fun getAtmosphereTest() {
```

Comment:
probably rename to `removeAtmosphereTest()`.

Reply:
yes. good point